renovate.json

```diff
@@ -9,12 +9,17 @@
     {
       "matchPackagePatterns": ["^@angular/"],
       "groupName": "angular"
+    },
+    {
+      "matchPackageNames": ["^@ionic/"],
```

Comment:
I'm not real familiar with renovate but [`matchPackageNames`](https://docs.renovatebot.com/configuration-options/#matchpackagenames) says:

> `matchPackageNames`: Use this field if you want to have one or more exact name matches in your package rule. See also `excludePackageNames`. Example: `"matchPackageNames": ["angular"],`

Should this also be [`matchPackagePatterns `](https://docs.renovatebot.com/configuration-options/#matchpackagepatterns)? 

> `matchPackagePatterns`: Use this field if you want to have one or more package names patterns in your package rule. See also `excludePackagePatterns`. Example: `"matchPackagePatterns": ["^angular"],`

Reply:
Yes! Good catch, updated here: https://github.com/ionic-team/ionic-docs/pull/3030/commits/abb75395213714abd2e23626901925e29ed6502f